.github/workflows/feature-branch-auto-merge.yml

```
@@ -0,0 +1,34 @@
+name: Feature branch auto merge
+
+permissions: read-all
+
+on:
+  push:
+    branches:
+      - main  # Set a branch to deploy
+      
+env:
+
+  MY_REPO: https://GoogleCloudPlatform:${{secrets.GITHUB_TOKEN}}@github.com/GoogleCloudPlatform/magic-modules.git
+
+  MY_BRANCH: feature-branch-release-5.0.0
```

Could we use a more descriptive name like `FEATURE_BRANCH`?

```suggestion
  FEATURE_BRANCH: feature-branch-release-5.0.0
```